### DISCUSSION TITLE
Issue with `prisma migrate deploy`

### DISCUSSION THREAD

**Discussion by emrysal** [prisma/prisma#8756] [Q&A] [answered]
Hi Everyone,

I'm experiencing an issue with Prisma throwing an error. When I run `prisma migrate dev` for a reason I can't explain I receive a div of all the FK's and Unique keys that already exist within my PostgreSQL DB. This is Prisma 2.26.0. What could be the issue here? I'm at a loss and don't know where to start looking.

Edit: This may be related: https://github.com/prisma/prisma/issues/8124

```
Error: P3005

The database schema for `127.0.0.1:5432` is not empty. Read more about how to baseline an existing production database: https://pris.ly/d/migrate-baseline
```

```
[*] Changed the `Attendee` table
  [+] Added foreign key on columns (bookingId)

[*] Changed the `Availability` table
  [+] Added foreign key on columns (eventTypeId)
  [+] Added foreign key on columns (userId)

[*] Changed the `Booking` table
  [+] Added unique index on columns (uid)
  [+] Added foreign key on columns (eventTypeId)
  [+] Added foreign key on columns (userId)

[*] Changed the `BookingReference` table
  [+] Added foreign key on columns (bookingId)

[*] Changed the `Credential` table
  [+] Added foreign key on columns (userId)

[*] Changed the `EventType` table
  [+] Added foreign key on columns (userId)

[*] Changed the `EventTypeCustomInput` table
  [+] Added foreign key on columns (eventTypeId)

[*] Changed the `Membership` table
  [+] Added foreign key on columns (teamId)
  [+] Added foreign key on columns (userId)

[*] Changed the `SelectedCalendar` table
  [+] Added foreign key on columns (userId)

[*] Changed the `VerificationRequest` table
  [+] Added unique index on columns (identifier, token)
  [+] Added unique index on columns (token)

[*] Changed the `users` table
  [+] Added unique index on columns (email)
```

**Comment by emrysal** [comment, marked as answer]
Due to an unknown reason my `_prisma_migrations` table disappeared without a trace. Resolved by reapplying all migrations with `npx prisma resolve --applied <migration>`

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
